server/db/db.js

```diff
@@ -1,8 +1,16 @@
-import CONFIG from '../config.js';
-//TODO: depedencies
-//TODO: signal flow for SQL vs No-SQL (TBD - after seed)
+const CONFIG = require('../../config.js');
+const pg = require('pg');
+const path = require('path');
 
```

Comment:
Looks fine; I can't see the original config file so I wonder if it would be possible to just input the entire config into the Client constructor?

On the other hand, this version has very good readability.